runelite-client/src/main/java/net/runelite/client/plugins/bank/BankPlugin.java

```diff
@@ -126,6 +126,12 @@ public void keyPressed(KeyEvent e)
 					return;
 				}
 
+				Widget chatboxFullInput = client.getWidget(WidgetInfo.CHATBOX_FULL_INPUT);
+				if (chatboxFullInput != null && !chatboxFullInput.isHidden())
```

Comment:
Not sure if this change is correct in general, but, this fails if run with `-ea` due to being called off the client thread.